src/components/Modals/Nfts/NftModal.tsx

```diff
@@ -75,14 +75,9 @@ export const NftModal: React.FC<NftModalProps> = ({ nftItem }) => {
   const [isLargerThanMd] = useMediaQuery(`(min-width: ${breakpoints['md']})`, { ssr: false })
   const accountIds = useAppSelector(selectWalletAccountIds)
 
-  const collectionAddresses = useMemo(
-    () => (nftItem.collection.id ? [fromAssetId(nftItem.collection.id).assetReference] : []),
-    [nftItem?.collection?.id],
-  )
-
-  const { data: nftCollection } = useGetZapperCollectionsQuery(
-    { accountIds, collectionAddresses },
-    { skip: !collectionAddresses?.length },
+  const { data: nftCollection } = useGetZapperCollectionQuery(
```

Comment:
Renamed and args changed - this really is getting a single collection data and was turbo confusing - note this is currently rugged upstream and probably needs to be ditched altogether for now until Zapper fix this / we implement alchemy in the follow-up PR

We will also want to make this eventually exhaust different APIs, similar to what we do for user balances